webhook.rst

```diff
@@ -16,8 +20,75 @@ Installation
 
     $ composer require symfony/webhook
 
+
+Consuming Webhooks
+------------------
+
+Consider a third-party API that allows you to track stock levels for various products and can send webhooks to your
+Symfony application.
+
+From the perspective of your application (the *consumer*), which receives the webhook, three primary phases need to be anticipated:
+
+1) Receiving the webhook
+
+2) Verifying the webhook and constructing the corresponding Remote Event
+
+3) Using the received data.
+
+Symfony Webhook, when used alongside Symfony RemoteEvent, streamlines the management of these fundamental phases.
+
+A Single Entry Endpoint: Receive
+--------------------------------
```

Comment:
shouldn't this be a sub-heading of the `Consuming Webhooks` section instead ? With the new doc covering both consuming and providing webhooks, I think we should have only 3 `h2` titles: `Installation`, `Consuming webhooks` and `Providing webhooks`